### PR TITLE
Update start_linux.sh

### PR DESCRIPTION
**Pull request by 0xb4dc0d3x**
added auto install dependencies of index.py on linux

### CODE REVIEW

**Comment by AlexFlipnote** [comment]
Don't want people to forcefully install every time

**Comment by 0xb4dc0d3x** [comment]
shall I use inputs for installing dependencies?

**Comment by ZackeryRSmith** [comment]
> Don't want people to forcefully install every time

I agree, it also creates a big wall of text (which can be hidden by running pip in silent mode)

You also don't want to ask the user if they want to install dependencies every time they run `start_linux.sh`, that'd just be irritating! The README goes over how to install the dependencies. No need to have it done in the start script.

**Comment by 0xb4dc0d3x** [comment]
ok

File: start_linux.sh [hunk 1]
```diff
@@ -21,7 +21,9 @@ if !command -v python3 &> /dev/null; then
     exit
 fi
 
+pip3 install -r requirements.txt
```
Comment (ZackeryRSmith) [inline start_linux.sh:24]: pip should be invoked via `python3` not using `pip3`. As `pip3` doesn't always point to the correct python install you want to use. Thus, to make sure you are installing the dependencies to the proper location do `python3 -m pip install -r requirements.txt`

Comment (0xb4dc0d3x) [reply start_linux.sh:24]: ok

**Pull request closed without merging**